index.html

```diff
@@ -0,0 +1,219 @@
+    <a>LET'S CONNECT</a>
+    <ul class="socialnetworks">
+    <li>
+        <a href="https://twitter.com/home">
+            <img src="twit.svg" alt="Twiter">
+        </a>
+    </li>
+    <li>
+        <a href="https://www.linkedin.com/feed/">
+            <img src="ln.svg" alt="Linkedin">
+        </a>
+    </li>
+    <li>
+        <a href="https://medium.com/">
+            <img src="m.svg" alt="Medium">
+        </a>
+    </li>
+    <li>
+        <a href="https://github.com/djo1975">
+            <img src="cat.svg" alt="Github">
+        </a>
+    </li>
+    <li>
+       <a href="https://github.com/djo1975">
+            <img src="v.svg" alt="Victory">
+        </a>
+    </li>
+    </ul>
+      <section class="my-works">
+        <div class="container">
+            
+          <div class="items">
+            <div class="portofolio-img"></div>
+            <h2 class="project-title">Tonic</h2>
+              <div class="c-b-2015">
+              <h6>CANOPY</h6>
+              <ul class="c-b-2015-ul">
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+  
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="prog-lang">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a>
+  
+          </div>
+        </section>
+          <section class="my-works">
+            <div class="container">
+              <div class="items">
+                <div class="portofolio-img"></div>
+                <h2 class="project-title">Tonic</h2>
+                  <div class="c-b-2015">
+                  <h6>CANOPY</h6>
+                  <ul class="c-b-2015-ul">
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+      
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="prog-lang">
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javaScript</li>
+                </ul>
+                <a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a>
+              </div>
+                <div class="items">
+                  <div class="portofolio-img-2"></div>
+                  <h2 class="project-title">Multi-Post <br> Stories</h2>
+                    <div class="c-b-2015">
+                    <h6>CANOPY</h6>
+                    <ul class="c-b-2015-ul">
+                      <li>Back End Dev</li>
+                      <li>2015</li>
+                    </ul>
+                  </div>

```

Comment:
- Good job aligning the content of your project card with the `flex` display property, but there are some differences that still need to be addressed. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
-- The `see project` button should be below the project description text, please fix this for all your project cards 👍 👍 
-- The list of technologies should also be below the project description text, please fix this for all your project cards 🙏 🙏 
<img width="148" alt="wgawrgaw" src="https://user-images.githubusercontent.com/73607512/205452847-4a0d5e29-eaef-4b34-86e7-6a4c8b32225a.PNG">

Figma Design:
<img width="148" alt="wgawrgaw" src="https://user-images.githubusercontent.com/73607512/205452943-fb919927-c1f5-4014-90b8-3b85a87bc32d.PNG">